client/lib/main.dart

```diff
@@ -37,6 +38,10 @@ PackageInfo _packageInfo;
 
 PackageInfo get packageInfo => _packageInfo;
 
+// ATTENTION: never check this in as 'true'! Always set back to 'false' before
+//            sending out your PR. This is verified by `test/firebase_test.dart`.
```

Comment:
`test/firebase_test.dart` looks like a file but I don't see it in this PR? Good to have a test for mistakes like this.

Reply:
Oops! Added the missing file.